primitives/src/sequence.rs

```diff
@@ -16,17 +16,17 @@
 //! [BIP-125]: <https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki>
 
 use core::fmt;
-
```

Comment:
FYI we have `rustfmt` configured to keep a space between `core`/`std` and external crate imports and `crate::foo`.  See `rustfmt.toml`.

```
group_imports = "StdExternalCrate" # Default "Preserve"
```

Reply:
ah my bad, didn't know about that. I'll undo this along with some other changes